core/utils/cuckoo_map.h

```diff
@@ -180,7 +180,7 @@ class CuckooMap {
   Entry* Find(const K& key, const H& hasher = H(), const E& eq = E()) {
     // Blame Effective C++ for this
     return const_cast<Entry*>(
-        static_cast<const typeof(*this)&>(*this).Find(key, hasher, eq));
+        static_cast<const CuckooMap&>(*this).Find(key, hasher, eq));
```

Comment:
I'm not entirely sure how `typeof` works, but I guess it just statically obtains the type name of its argument? In this case, the type of `*this` would be `CuckooMap` anyway?